packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -496,177 +496,187 @@ export default class TableResize implements EditorPlugin {
+                                cell.td.style.wordBreak = 'break-word';
+                                cell.td.style.whiteSpace = 'normal';
+                                cell.td.style.width = `${newWidth}px`;
+                            }
+                        }
+
+                        if (shouldResizeY) {
+                            // the height of some external table is fixed, we need to make it resizeable
+                            vtable.table.removeAttribute('height');
+                            vtable.table.style.height = null;
+                            if (j == 0) {
+                                const originalHeight = cell.td.style.height
                                     ? parseFloat(
-                                          cell.td.style.width.substr(
+                                          cell.td.style.height.substr(
                                               0,
-                                              cell.td.style.width.length - 2
+                                              cell.td.style.height.length - 2
                                           )
                                       )
-                                    : cell.td.getBoundingClientRect().right -
-                                      cell.td.getBoundingClientRect().left;
-                                const newWidth = originalWidth * ratioX;
-                                cell.td.style.boxSizing = 'border-box';
-                                if (newWidth >= MIN_CELL_WIDTH) {
-                                    cell.td.style.wordBreak = 'break-word';
-                                    cell.td.style.width = `${newWidth}px`;
-                                }
-                            }
-
-                            if (shouldResizeY) {
-                                if (j == 0) {
-                                    const originalHeight =
-                                        cell.td.getBoundingClientRect().bottom -
-                                        cell.td.getBoundingClientRect().top;
-                                    const newHeight = originalHeight * ratioY;
-                                    if (newHeight >= MIN_CELL_HEIGHT) {
-                                        cell.td.style.height = `${newHeight}px`;
-                                    }
-                                } else {
-                                    cell.td.style.height = '';
+                                    : cell.td.getBoundingClientRect().bottom -
+                                      cell.td.getBoundingClientRect().top;
+                                const newHeight = originalHeight * ratioY;
+                                if (newHeight >= MIN_CELL_HEIGHT) {
+                                    cell.td.style.height = `${newHeight}px`;
                                 }
+                            } else {
+                                cell.td.style.height = null;
```

Comment:
What if there is merged cells in table?

Reply:
What if the first column/row has merged cells?

Reply:
And if we only set width/height for first column/row, what if user then delete the first column/row?

Reply:
![fistcell](https://user-images.githubusercontent.com/4762468/121634713-ef335f00-ca39-11eb-851a-57ab03b57e14.gif)

Is this what you mean?

Reply:
if use deleted the first column, we will recalculate the first column and it's still working. For rows, we recalculate every cell.  Please free feel to test

Reply:
Can you point me the code to recalculate?

Reply:
I didn't see issue here, just want to see how it is done in the code.